brainfuck/bf.rs

```diff
@@ -90,7 +90,7 @@ impl<'a> Printer<'a> {
         }
     }
 
-    fn get_checksum(&self) -> i32 {
+    const fn get_checksum(&self) -> i32 {
```

Comment:
I don't see that function used in the const context, am I missing something here?

Reply:
Yeah, currently nowhere can benefit of it.